### PR TITLE
Add empty cassandra_benches file

### PR DESCRIPTION
**Pull request by conorbros**
<s>- Move the init code out of the actual bench-marking code</s>
<s>- Move each of the Redis benchmarks to functions so they can be run individually without having to run other benchmark's init code beforehand.</s>

- add cassandra_benches.rs file so CI on #512 works

### CODE REVIEW

File: shotover-proxy/benches/redis_benches.rs
Comment (rukai) [inline shotover-proxy/benches/redis_benches.rs:11]: If this helps the noise issue then we should drop the noise_threshold down to a suitable value

Comment (conorbros) [reply shotover-proxy/benches/redis_benches.rs:11]: Yeah, I'm thinking around 5-10%?

Comment (rukai) [reply shotover-proxy/benches/redis_benches.rs:11]: Looking at https://github.com/shotover/shotover-proxy/runs/5104088154?check_suite_focus=true 20% looks about right.
We can bump it up another 5-10% if we get false positives.

Comment (conorbros) [reply shotover-proxy/benches/redis_benches.rs]: Alright, changed to 20%.

**Comment by rukai** [comment]
I tested locally and this doesnt fix the initialization problem.
You can clearly see this by running something like `cargo bench null` and seeing all the shotover initialization log spam.

**Comment by rukai** [comment]
You've made a good discovery that the noise issue is made drastically worse by my init hack, but in order to land this PR we should not regress on running individual benchmarks, instead we should add the ability to specify benchmark init logic to criterion.

**Comment by conorbros** [comment]
I think that since the noisy benchmarks don't give us any meaningful data, we should land this and open a ticket for adding that functionality to criterion.

Commenting out benchmarks to run individually is not ideal, but I think it's better than not getting anything valuable from the benchmarks.

**Comment by rukai** [comment]
Criterion is undermaintained and just opening a ticket wont get us anywhere.
I'm happy for this PR to revert the init hack if someone immediately starts working on implementing init functionality in criterion. I guess I'll be that person and start working on it now.

**Comment by conorbros** [comment]
Thanks! I'll move back to the original setup and then wait until that your upcoming PR to criterion is merged.

**Review by rukai — APPROVED** [review]

**Review by benbromhead — APPROVED** [review]

**Pull request merged**
